.github/workflows/sonarqube.yml

```
@@ -2,11 +2,9 @@ name: SonarQube Analysis
 
 on:
   push:
-    branches:
-      - main
+    branches: [test-deployment-do, deployment-digitalocean]
   pull_request:
-    branches:
-      - main
+      branches: [test-deployment-do, deployment-digitalocean]
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 6

(indentation)